ppr-ui/src/interfaces/mhr-registration-interfaces/MhrRegistrationIF.ts

```diff
@@ -36,7 +36,7 @@ export interface MhrRegistrationIF {
 
 export interface NewMhrRegistrationApiIF {
   draftNumber?: string
-  documentId: string
+  documentId?: string
```

Comment:
Why do we need to make it optional?

Reply:
The ticket says no Document ID for Manufacturer.

I can make it optional or populate it with any empty string I guess.
Do you have any insights on which approach is better?

Reply:
Since it's just one field, I'm not too opposed - we can make it optional here, instead of using a new (dedicated) interface for Manufacturers' Registrations. If we end up modifying too much, we'll create a new IF.

Reply:
Agreed, I had the same thought process!